substrate/runtime/consensus/src/lib.rs

```diff
@@ -96,6 +100,14 @@ decl_module! {
 	}
 }
 
+decl_storage! {
+	trait Store for Module<T: Trait> as Consensus {
+		// Authorities set actual at the block execution start. IsSome only if
```

Comment:
`IsSome` or `!empty()`? (with `default` you don't get access to whether it's `Some` or `None`)